pkg/workflow_execution.go

```diff
@@ -408,6 +410,140 @@ func (c *Client) createWorkflow(namespace string, workflowTemplateID uint64, wor
 		return nil, err
 	}
 
+	newTemplateOrder := []wfv1.Template{}
+	for tIdx, t := range wf.Spec.Templates {
+		if t.Metadata.Labels != nil {
+			if sidecar, ok := t.Metadata.Labels["sidecar"]; ok {
+				if sidecar == "sys-tensorboard-like" {
```

Comment:
we should probably call this `sys-visualization-tool` or something similar to be more generic.